src/valora-dapp-list.json

```diff
@@ -394,6 +394,15 @@
       "url": "https://core.allbridge.io/",
       "listOnAndroid": true,
       "listOnIos": true
+    },
+    {
+      "name": "Merkle",
+      "id": "merkle-angle",
+      "categoryId": "lend-borrow-earn",
+      "categories": ["lend", "borrow", "earn"],
+      "url": "https://merkl.angle.money/",
```

Comment:
I saw https://app.merkl.xyz/ is another way to reach the same dapp.
I wonder which one they consider they'd recommend.

Reply:
I guess according to https://docs.merkl.xyz/, https://app.merkl.xyz/ should be preferred?

Reply:
Fixed in f5a973534f43b668b54f71252d361f4cab278f31!